common/psbt_open.c

```diff
@@ -118,8 +105,8 @@ static const u8 *linearize_output(const tal_t *ctx,
 	/* We don't care if the keypaths change */
 	psbt->outputs[0].keypaths.num_items = 0;
```

Comment:
Agreed.  It won't *in fact* leak because we use tal on all our libwally allocations (so it will eventually be freed with the psbt itself), but it's fugly.